samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/readme.md

```diff
@@ -0,0 +1,162 @@
+# Get started sending and receiving messages from ServiceBus queues using QueueClient
+
+In order to run the sample in this directory, replace the following bracketed values in the `Program.cs` file.
+
+```csharp
+const string ServiceBusConnectionString = "{ServiceBus connection string}";
```

Comment:
Do you want to add a comment on how to obtain these things? The samples are for complete newbies and might not know where to find them. Just a quick note on where to find it (on the portal)?